ci.hocon

```diff
@@ -181,13 +181,39 @@ graal-vm-snapshot: {
   }
 }
 
+jruby-benchmark: {
+  setup: ${common-base.prelude}
+
+  downloads: {
+    JRUBY_HOME: {name: jruby, version: "9.1.12.0"}
+  }
+
+  environment: {
+    HOST_VM: server,
+    HOST_VM_CONFIG: default
+    GUEST_VM: jruby,
+    GUEST_VM_CONFIG: indy,
+    RUBY_BIN: "$JRUBY_HOME/bin/jruby",
+    JT_BENCHMARK_RUBY: "$JRUBY_HOME/bin/jruby",
```

Comment:
Why do we have two of these now?

Reply:
`jt benchmark` honors `JT_BENCHMARK_RUBY`, but `jt ruby` honors `RUBY_BIN`. For whatever reason, the server benchmarks are written using `jt ruby`, not `jt benchmark`. I don't believe we've ever run the server benchmarks on anything other than TruffleRuby, thus the new addition.